admin/view/template/extension/payment/razorpay_plan_form.twig

```diff
@@ -0,0 +1,201 @@
+{{ header }}{{ column_left }}
```

Comment:
move all the subscription related files under `**subscription**` folder. Follow the same for other locations as well.